src/components/organisms/SiteHeader/index.tsx

```diff
@@ -77,6 +78,13 @@ const otherToolsItems: Array<IOtherToolsItem> = [
     icon: <CmlIcon width="24" height="24" />,
     description: 'Open-source CI/CD for ML projects',
     href: '/'
+  },
```

Comment:
Does the Other Tools dropdown look too tall now?

![image](https://user-images.githubusercontent.com/43496356/145623009-42cd5b59-61e2-4645-8714-11b4c7b5cea8.png)

If so we could:

* just close this pr and wait till we can get an updated design that makes room for more items
* take this off and just add a link to the footer
* take off the CML link in the Other Tools? Though that would only fix things temporarily until we get another tool like LDB.

Reply:
I think it's on the cusp of being too large, but is acceptable as-is. We'll definitely need to figure something out before adding another tool here.

I'd imagine it would be having multiple tools per line, similarly to how Google does:

![Screenshot of Google's products menu](https://user-images.githubusercontent.com/9111807/145630541-1e407744-6c97-45c7-a03c-4f0f3b634ee6.png)